src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -36,7 +36,7 @@ object ConsumptionPlan : Plan(
                     StorageKeyParser.parse("reference-mode://{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/Thing}{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/!:writingArcId/handle/my-handle-id}"),
                     HandleMode.Read,
                     EntityType(Reader_Data_Spec.SCHEMA),
-                    null
+                    Ttl.Infinite
```

Comment:
may be a follow up PR, but it would be nice to add a ttl to the test recipe as well.

Reply:
This is a good idea; I'll update our example recipe.